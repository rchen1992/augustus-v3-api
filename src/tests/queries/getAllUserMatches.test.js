const mockMatchRepo = require('@tests/mockRepos/match');
const { testQuery } = require('@tests/utils');

testQuery({
    id: "Get all of a user's matches",
    query: `
        {
            user(id: 1) {
                id
                matches {
                    id
                }
            }
        }
    `,
    variables: {},
    context: () => {
        return {
            repos: {
                match: mockMatchRepo,
            },
        };
    },
    expected: undefined,
});
