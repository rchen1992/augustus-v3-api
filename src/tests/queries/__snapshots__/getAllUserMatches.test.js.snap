// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`query: Get all of a user's matches 1`] = `
Object {
  "data": Object {
    "user": Object {
      "id": "1",
      "matches": Array [
        Object {
          "id": "1",
        },
        Object {
          "id": "5",
        },
      ],
    },
  },
}
`;
